recipes/commonmark/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "CommonMark" %}
+{% set version = "0.7.2" %}
+{% set sha256 = "3cf8971d85b5782563ef8a3cc9c9ec92f09f4e6b856e42b0cbf1931927218e1b" %}
+
+package:
+  name: {{ name | lower }}
+  version: {{ version }}
+
+source:
+  fn:  {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - cmark = CommonMark.cmark:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - future
+
+test:
+  imports:
+    - CommonMark
+    - CommonMark.render
+    - CommonMark.tests
+
+  commands:
+    - cmark --help
+
+  requires:
+    - flake8
```

Comment:
It came up in the automatic conda skeleton-generated script, so I left them in. I can take them out though.